src/plugins/spaces.rs

```diff
@@ -0,0 +1,9 @@
+pub struct Spaces;
+
+impl crate::core::Plugin for Spaces {
+    regex!(r"\(|\)", r"\[|\]", r"[-+*/%^&|<>=]");
```

Comment:
you need to define them differently - through NOT
everything that is not literals, variables and brackets are operators